<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>570</width>
    <height>451</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>串口小助手</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>串口操作</string>
     </property>
     <widget class="QPushButton" name="btnCheck">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>串口检测</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClearRec">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>清空接收</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpen">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>串口打开</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClearSend">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>清空发送</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbBoxPort">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbBoxBaud">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbBoxDataBits">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbBoxParity">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbBoxStopBits">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>100</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>串口号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>比特率：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>数据位：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>停止位：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>奇偶校验：</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioBtnChar">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>100</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>字符显示</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioBtnHex">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>100</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Hex显示</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelStatus">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px;
background-color:rgb(170,0,0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>数据接收</string>
     </property>
     <widget class="QTextEdit" name="textEditRec">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>561</width>
        <height>91</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>数据发送</string>
     </property>
     <widget class="QTextEdit" name="textEditSend">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>20</y>
        <width>351</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioBtnChar2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>40</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>字符发送</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioBtnHex2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>70</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Hex发送</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnSend">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>数据发送</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="timeSend">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>105</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>定时发送</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>130</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
